views/courses/course.hbs

```diff
@@ -68,8 +68,13 @@
           <button class="tab" data-testid="topics" data-tab="js-topics">
             <span class="text">{{$t "courses._course.tab_label.topics"}}
           </button>
-          <button class="tab" data-testid="hometasks" data-tab="js-homeworks">
-            <span class="text">{{$t "courses._course.tab_label.tasks"}} <span class="count-badge" data-badge="{{ add (arrayLength ../myhomeworks)  (arrayLength ../homeworks) }}"/></span>
+          {{#if isTeacher}}
+            <button class="tab" data-testid="hometasks" data-tab="js-homeworks">
```

Comment:
You can place `<button class="tab" data-testid="hometasks" data-tab="js-homeworks">` above the conditional statement